layers/+lang/bibtex/README.org

```diff
@@ -82,3 +101,26 @@ key binding added:
 | Key binding | Description     |
 |-------------+-----------------|
 | ~SPC m i c~ | Insert citation |
+
+*EBIB*
+
+*Index buffer*
+| Key Binding | Description                                           |
+|-------------+-------------------------------------------------------|
+| ~J/K~       | scroll page down/up                                   |
+| ~g j~       | jump to entry (searches only author and title fields) |
```

Comment:
you can also use `spacemacs|define-jump-handlers`. then user only needs to press `gd` and you don't need to manually bind the key.

(But the downside is it won't be able to receive universal argument. So you still need to manually bind `SPC u g j`.

Reply:
It is a nice suggestion (also because I learn things), but here the jump is more an index search feature than a jump to definition feature (similar to swoop/swiper). It is not a jump to definition, because, I would argue that, that should jump to the .bib entry, while this jump only jumps to the 'index' entry in the index buffer. And as you mentioned already, it would be strange to have another, different, binding for the 'universal argument' only.